google/cloud/security/common/gcp_api/bigquery.py

```diff
@@ -64,10 +64,12 @@ def get_bigquery_projectids(self):
 
         paged_results = self._build_paged_result(
             request, bigquery_projects_api, self.rate_limiter)
+
+        flattened_result = self._flatten_list_results(paged_results, key)
+                
         project_ids = []
-        for page in paged_results:
-            for project in page.get('projects', []):
-                project_ids.append(project.get('id'))
+        for result in flattened_result:
```

Comment:
this is super nitpicky, and I think we are probably not consistent about it, but does it make sense to make this (and the other for loop) a list comprehension? you don't have to, if you don't want to.